docs/rp/integration/wallets.md

```diff
@@ -0,0 +1,608 @@
+# Integration Guide for Wallets
+
+## Purpose of this document
+
+This document serves as a guide for integrating end-user
+[wallets](https://github.com/hyperledger/aries-rfcs/blob/master/concepts/0004-agents/README.md) to the RP Adapter.
+
+
+## Overview
+
+The End User's web browser is redirected to the RP Adapter by the Relying Party. The Adapter then 
+requests [web credentials](https://w3c-ccg.github.io/credential-handler-api/) from the User's Agent. The request is
+in the form of a
+[Presentation Definition](https://identity.foundation/presentation-exchange/#presentation-definition). A
+[DIDComm connection](https://github.com/hyperledger/aries-rfcs/blob/master/concepts/0005-didcomm/README.md) is also
+bootstrapped via an
+[Out-of-Band invitation](https://github.com/hyperledger/aries-rfcs/blob/master/features/0434-outofband/README.md).
+
+The Adapter may request verifiable credentials with statements about a subject, or it may request
+[Authorization Credentials](https://github.com/trustbloc/context/blob/master/vc/authorization-credential-v1.jsonld) that
+the RP Adapter can then use to fetch the "normal" credentials.
+
+> **TODO** document authorization credentials
+
+In all cases, once the RP Adapter has collected all credentials and verified requirements are met as per the
+original presentation definition, the End User's web browser will be redirected back to the Relying Party's origin.
+
+![overview](http://www.plantuml.com/plantuml/proxy?src=https://raw.githubusercontent.com/trustbloc/edge-adapter/master/docs/rp/integration/wallet_rpadapter_overview.puml)
```

Comment:
Diagram will render properly once PR is merged. Here is the preview:

![image](https://user-images.githubusercontent.com/2019896/97241951-ec486e80-17c8-11eb-8df2-5d67f6933040.png)